contracts/WitnetRequestBoard.sol

```diff
@@ -0,0 +1,258 @@
+ * The result of the requests will be posted back to this contract by the bridge nodes too.
+ * @author Witnet Foundation
+ */
+contract WitnetRequestBoard is WitnetRequestBoardInterface {
+    // TODO: update max report result gas value
+    uint256 public constant ESTIMATED_REPORT_RESULT_GAS = 102496;
+
+    struct DataRequest {
+        address requestAddress;
+        address requestor;
+        uint256 drOutputHash;
+        uint256 reward;
+        uint256 gasPrice;
+        bytes result;
+        uint256 drHash;
+    }
+
+    // Owner of the Witnet Request Board
+    address public witnet;
+
+    // List of addresses authorized to report data request
+    address[] public committee;
+
+    // Witnet Requests within the board
+    DataRequest[] public requests;
+
+    // Event emitted when a new DR is posted
+    event PostedRequest(address indexed _from, uint256 _id);
+
+    // Event emitted when a result proof is posted
+    event PostedResult(address indexed _from, uint256 _id);
+
+    // Only the committee defined when deploying the contract should be able to report results
+    modifier isAuthorized() {
+        bool senderAuthorized = false;
+        for (uint256 i; i < committee.length; i++) {
+            if (committee[i] == msg.sender) {
+                senderAuthorized = true;
+                break;
+            }
+        }
+        require(senderAuthorized == true, "Sender not authorized");
+        _;
+    }
+
+    // Ensures the result has not been reported yet
+    modifier resultNotIncluded(uint256 _id) {
+        require(requests[_id].result.length == 0, "Result already included");
+        _;
+    }
+
+    // Ensures the request has not been manipulated
+    modifier validDrOutputHash(uint256 _id) {
+        require(
+            requests[_id].drOutputHash ==
+                uint256(
+                    sha256(Request(requests[_id].requestAddress).bytecode())
+                ),
+            "The dr has been manipulated and the bytecode has changed"
+        );
+        _;
+    }
+
+    // Ensures the request id exists
+    modifier validId(uint256 _id) {
+        require(requests.length > _id, "Id not found");
+        _;
+    }
+
+    /// @notice Initilizes a centralized Witnet Request Board with an authorized committee.
+    /// @param _committee list of authorized addresses.
+    constructor(address[] memory _committee) public {
+        witnet = msg.sender;
+        committee = _committee;
+        // Insert an empty request so as to initialize the requests array with length > 0
+        DataRequest memory request;
+        requests.push(request);
+    }
+
+    /// @dev Posts a data request into the WRB in expectation that it will be relayed and resolved in Witnet with a total reward that equals to msg.value.
+    /// @param _requestAddress The request contract address which includes the request bytecode.
+    /// @return The unique identifier of the data request.
```

Comment:
In other parts of the code you changed unique by secuential

Reply:
I change `unique` for `sequential` as there were more occurrences